deploy/kubectl/openebs-nfs-provisioner.yaml

```diff
@@ -155,8 +155,8 @@ metadata:
     cas.openebs.io/config: |
       - name: NFSServerType
         value: "kernel"
-      #- name: BackendStorageClass
-      #  value: "openebs-hostpath"
+      - name: BackendStorageClass
+        value: "openebs-hostpath"
```

Comment:
can we add an example with `FSGID` here with comment. why are we commenting out this section?

Reply:
> why are we commenting out this section?

For testing purpose, I uncommented but forgot to comment back. I will comment back along with FSGID value